### PR TITLE
fixed the pack_known_words_functionality

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6521

## Description
Fix the behavior of the --use-pack-known-words parameter

### CODE REVIEW

**Review by thefrieddan1 — APPROVED** [review]
nice

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -2894,8 +2894,8 @@ def test_content(ctx, **kwargs):
     default=False,
 )
 @click.option(
-    "-pkw",
-    "--use-packs-known-words",
+    "-pkw/ -npkw",
+    "--use-packs-known-words/ --no-use-packs-known-words",
```
Comment (dorschw) [inline demisto_sdk/__main__.py:2898]:
```suggestion
    "--use-packs-known-words/--skip-packs-known-words",
```
please change elsewhere as well

**Review by dorschw — CHANGES_REQUESTED** [review]

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (omerKarkKatz) [reply demisto_sdk/__main__.py]: Done

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -2894,8 +2894,8 @@ def test_content(ctx, **kwargs):
     default=False,
 )
 @click.option(
-    "-pkw",
-    "--use-packs-known-words",
+    "-pkw/ -npkw",
+    "--use-packs-known-words/ --skip-packs-known-words",
```
Comment (dorschw) [inline demisto_sdk/__main__.py:2898]: does it worth with a space after the `/`?

**Review by dorschw — APPROVED** [review]

File: demisto_sdk/__main__.py [hunk 3]
```diff
@@ -2894,8 +2894,8 @@ def test_content(ctx, **kwargs):
     default=False,
 )
 @click.option(
-    "-pkw",
-    "--use-packs-known-words",
+    "-pkw/ -spkw",
+    "--use-packs-known-words/ --skip-packs-known-words",
```
Comment (dorschw) [inline demisto_sdk/__main__.py:2898]: does it work with a space after the `/`?

Comment (omerKarkKatz) [reply demisto_sdk/__main__.py]: not sure but removed the space for any case..

**Pull request merged**